### DISCUSSION TITLE
Any chance of a new release?

### DISCUSSION THREAD

**Discussion by jrd-org** [poweradmin/poweradmin#404] [Q&A]
2.1.7 is getting a bit long in the tooth.  Trying to install it on a new server, I first found the PHP 8 incompatibility.  I backed off to PHP 7, but then found the mcrypt incompatibility.  At least the latter appears to have been fixed already in mainline.

I'd love to keep using poweradmin, it's simple and does the job.  It'd be easier if there was an updated package available.

I'm willing to help with testing . . .

**Comment by edmondas** [comment]
In the coming weeks, I plan to review existing pull requests and address any critical issues, then perform some manual testing. Thus, a new version can be expected soon.

**Comment by edmondas** [comment]
@jrd-org I encourage you to test the newly released version and report any bugs you find.

**Comment by jrd-org** [comment]
I've spent a little time with 2.1.8.  So far it looks great!  If I find any issues I'll file bug reports, or generate some PRs.  Thanks!
